src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -110,6 +114,11 @@ case class MethodRepoConfigurationQuery(
 
 case class ConflictingEntities(conflicts: Seq[String])
 
+case class WorkspaceListResponse(accessLevel: String,
+                                 workspace: Workspace,
```

Comment:
It depends on what the UI wants to show. I'd imagine that they'd want everything.